### ISSUE TITLE
Replicating 7.x's collapsable "Details" display

### ISSUE THREAD

**Issue by mjordan** [Islandora/documentation#1359]
Islandora 7.x's metadata display was wrapped in a collapsable div that allowed the user to expand/collapse:

Collapsed:

![lasers1](https://user-images.githubusercontent.com/403918/69353292-20b7c000-0c33-11ea-8e81-646a318836ea.png)

Expanded:

![lasers2](https://user-images.githubusercontent.com/403918/69353328-3200cc80-0c33-11ea-8be5-c666cc26c404.png)

In Islandora 8, the metadata fields are always exposed (similar to the expanded option above). Do we want to emulate 7.x's behavior in 8, or at least provide a built-in option to do so?

**Comment by mjordan** [comment]
I'll check out https://www.drupal.org/project/field_group to see if this does the trick.

**Comment by seth-shaw-unlv** [comment]
I was just about to recommend field_group for doing this. However, perhaps this should be a recipe instead of a default (to limit the default install weight)?

**Comment by mjordan** [comment]
Agreed. I'm trying to test it now but the composer install failed. Will resume testing/troubleshooting later.

**Comment by mjordan** [comment]
Field Group works well. Here is my field group (lower left, it's circled in red to make it more visible) closed by default:

![closed](https://user-images.githubusercontent.com/403918/69483037-6d3c0080-0dd7-11ea-8b78-b2e5a22b59ed.png)

Clicking on the "Metadata" label opens it:

![open](https://user-images.githubusercontent.com/403918/69483049-7dec7680-0dd7-11ea-9c1f-952e5c515fa2.png)

It a bit clunky to set up.  One thing is that it looks like admins will need to configure this separately for every display/view mode (default, PDFjs, OpenSeadragon, etc.):

![fieldgroupconfig](https://user-images.githubusercontent.com/403918/69483103-0c60f800-0dd8-11ea-9482-c72e3b50d901.png)

But once it's done, it's set and forget. It looks like the field group config is portable:

![fieldgroupyaml](https://user-images.githubusercontent.com/403918/69483132-6e216200-0dd8-11ea-8b0a-b65aadc20f68.png)

So maybe it could become part o the standard setup?

**Comment by mjordan** [comment]
Tagging @manez as this would be good candidate for a recipe.

**Comment by manez** [comment]
Finally got to this one on the tech call today, with a resounding "yes" for doing it as a recipe and not as a default.

**Comment by mjordan** [comment]
@manez, where do recipes fit into the DIG's current workplan?

**Comment by manez** [comment]
@mjordan https://github.com/Islandora-Labs/Islandora-Cookbook is still the home base for recipes. We've talked about evaluating whether those should get rolled into the main docs at some point, but that's pretty far ahead, and would build from what's in the github repo anyway.

**Comment by mjordan** [comment]
OK, thanks. I'll write up a recipe and open a PR on that repo.
